docs/Data Provider Nodes/data-provider-nodes-list.md

```diff
@@ -6,19 +6,19 @@ title: "Data Provider Nodes List"
 permalink: "docs/data-provider-nodes-list/"
 ---
 
-| Name                                                                      | Supported Blockchains                        |
-|:------------------------------------------------------------------------- |:-------------------------------------------- |
-| [Chainlink Alarm Clock (Testnet)](../chainlink-alarm-clock)               | Ethereum                                     |
-| [LCX (Testnet)](../lcx-testnet)                                           | Ethereum                                     |
-| [Kraken Rates Oracle Node](../kraken-rates-oracle-node)                   | Ethereum                                     |
-| [Tiingo EOD Stock Price Oracle](../tiingo-eod-stock-price-oracle)         | Ethereum, Binance Smart Chain                |
-| [GeoDB Oracle Node](../geodb-oracle-node)                                 | Ethereum, Binance Smart Chain                |
-| [TheRunDown Oracle Node](../therundown-oracle-node)                       | Ethereum, Polygon                            |
-| [DNS Ownership Oracle](../dns-ownership-oracle)                           | Ethereum, Binance Smart Chain, Polygon       |
-| [WatchSignals Luxury Watch Price Oracle](../watchsignals)                 | Ethereum, Binance Smart Chain                |
-| [Genesis Volatility Cryptocurrency Options Oracle](../genesis-volatility) | Ethereum, Binance Smart Chain, Polygon, xDAI |
-| [dxFeed Price Oracle](../dxfeed-oracle)                                   | Ethereum, Binance Smart Chain                |
-| [CipherTrace DeFi Compli Oracle](../ciphertrace-defi-compli-oracle)       | Ethereum, Binance Smart Chain, Polygon       |
-| [Bookmaker Ratings Oracle](../bookmaker-ratings-oracle)                   | Ethereum, Binance Smart Chain                |
-| [Finage Global Market Data Oracle](../finage-global-market-data-oracle)   | Ethereum, Binance Smart Chain                |
-| [SportMonks Sports Data Oracle](../sport-monks-oracle)                    | Ethereum, Binance Smart Chain                |
+| Name                                                                      | Data Category | Supported Blockchains                        |
+|:------------------------------------------------------------------------- |:------------- |:-------------------------------------------- |
```

Comment:
I think this would be more friendly if we moved the `Data Category` column to the left, then group and order by category.

Reply:
Do you mean having Data category, then Name, then Supported blockchains?

Reply:
Yes, and order by Data Category, alphabetically.